src/components/home/Home.js

```diff
@@ -1,29 +1,40 @@
+/* eslint-disable no-unused-vars */
 // eslint-disable-next-line no-unused-vars
```

Comment:
- [ ]  You have much linter disables in your code. Please, consider removing them - linters should help you to find the bugs on early steps of the development 👍